admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -87,32 +87,120 @@ You can verify that the Redis daemon is running with ``ps ax``::
  22203 ? Ssl    0:00 /usr/bin/redis-server 127.0.0.1:6379 
  
 Restart your Web server, add the appropriate entries to your ``config.php``, and 
-refresh your Nextcloud admin page. This example ``config.php`` configuration uses 
-Redis for the distributed server cache::
+refresh your Nextcloud admin page.
 
-  'memcache.distributed' => '\OC\Memcache\Redis',
-  'redis' => [
-       'host' => 'redis-host.example.com',
-       'port' => 6379,
-  ],
+Redis configuration in Nextcloud (config.php)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 For best performance, use Redis for file locking by adding this::
 
   'memcache.locking' => '\OC\Memcache\Redis',
 
+Further more, you can use Redis for the distributed server cache:: 
+
+  'memcache.distributed' => '\OC\Memcache\Redis',
```

Comment:
It's not furthermore, but the essential configuration

Reply:
Why essential? You could use _memcached_ for the distributed cache instead (`'memcache.distributed' => '\OC\Memcache\Memcached'`), and _Redis_ only for local cache. As such, _Redis_ is just one available option for the distributed cache. That's why I used "further more".

Reply:
After reading your next comment, it appears that using Redis for local cache only doesn't make much sense. I wonder if that would not need to be explained in some part of the document. I.e. recommended setups are:

- APCu for local cache
- Redis for distributed cache and file locking on a single server
- Redis cluster or Memcached cluster for muliti-server setups

Or something along those lines.

Reply:
> Why essential? You could use _memcached_ for the distributed cache instead (`'memcache.distributed' => '\OC\Memcache\Memcached'`), and _Redis_ only for local cache. As such, _Redis_ is just one available option for the distributed cache. That's why I used "further more".

For me it sounded that the `'memcache.distributed'` option is a nice to have.